src/hotspot/share/compiler/methodMatcher.cpp

```diff
@@ -269,6 +270,19 @@ void MethodMatcher::parse_method_pattern(char*& line, const char*& error_msg, Me
     c_match = check_mode(class_name, error_msg);
     m_match = check_mode(method_name, error_msg);
 
+    // Over-consumption
+    // method_name points to an option type or option name because the method name is not specified by users.
+    // In very rare case, the method name happens to be same as option type/name, so look ahead to make sure
+    // it doesn't show up again.
+    if ((OptionType::Unknown != parse_option_type(method_name) ||
+        CompileCommand::Unknown != CompilerOracle::string_to_option(method_name)) &&
+        strstr(line + bytes_read, method_name) == NULL) {
+      error_msg = "not specify any method pattern";
```

Comment:
"Did not specify any method pattern", perhaps?

Reply:
How about "Did not specify any method name" ?

Reply:
It was only the missing "Did " that got my attention. I guess "method name" works, too, but isn't it technically a pattern? At least it allows wildcards, such as `java.lang.Object::*`